scripts/build_llvm.sh

```diff
@@ -1,11 +1,18 @@
 #!/bin/bash
 
+#Contact: Stan Iwan
+#         Sofomo
+#         2024.03.19
+
 TIMESTAMP=$(date +"%Y%m%d_%H%M%S")
-LOG_FILE="${TIMESTAMP}_cmp_env_setup.log"
+LOG_FILE="${TIMESTAMP}_build_llvm.log"
 CURRENT_STEP=""
 COPY_BAREMETAL_TOOLCHAIN=false
 COPY_LINUX_TOOLCHAIN=false
 
+RISCV_GNU_TOOLCHAIN_COMMIT_SHA="9a28c809a5d355874def06a414aa5f272fae564d"
+LLVM_PROJECT_COMMIT_SHA="7718ac38a0c23597d7d02f0022eb89afe6d1b35f"
+
```

Comment:
this is duplicated below. changing one sha means changing all scripts that use it.

Use a central file to contain all hardcoded sha's 

scripts/sha.config

reference that file from bash.

#!/bin/bash
source $TOP/scripts/sha.config